modules/ROOT/nav.adoc

```diff
@@ -76,6 +76,9 @@ include::partial$nav-migration.adoc[]
  ** xref:reconnection-strategy-about.adoc[Reconnection Strategies]
   *** xref:reconnection-strategy-reference.adoc[Reconnection Strategy Reference]
  ** xref:sources-reference.adoc[Sources Reference]
+* xref:notifications-configuration-reference.adoc[Notifications Configuration Reference]
```

Comment:
Does it make sense to include this reference if the article that explains how to use notifications is not available? (/mule-server-notifications.adoc)

Reply:
It is added in this PR.

Reply:
The PR only lists 5 files: 
* nav.adoc
* configuring-mule-stacktraces.adoc
* notifications-configuration-reference.adoc
* reproducible-builds.adoc
* setting-environment-variables.adoc

May be you also added mule-server-notifications.adoc in your local branch but didn't get pushed in this PR?

Reply:
This is a reference to the  notifications-configuration-reference topic and not to the mule-server-notifications file, so I don't understand your comment.

Reply:
I am confused about your comment. The topic in question is the notifications-configuration-reference.adoc topic, which is in the PR.

Reply:
What I mean is that the notifications-configuration-reference.adoc article, is the doc reference of the mule-server-notifications.adoc article. The nav.adoc properly references the file you are adding, I'm just saying that that file contains the reference for the mule server notifications, which is not added yet.

Reply:
There is no broken link in the notification-configuration-reference.adoc article, so I don't really understand why we would not want to put this topic in. If we always wait for the file that refers to this file to go in, we will never finish. The material is still useful for users.